subprojects/docs/src/docs/userguide/maven_plugin.adoc

```diff
@@ -19,7 +19,7 @@
 [NOTE]
```

Comment:
:o: I think this `[NOTE]` should become a `[CAUTION]`. Otherwise it is easy to miss for the quick reader with the current formatting. If there is a corresponding place for ivy/uploadArchive, that should also become `[CAUTION]`.